src/modules/RF69/RF69.h

```diff
@@ -621,7 +621,7 @@ class RF69: public PhysicalLayer {
 
       \returns \ref status_codes
     */
-    int16_t setOOK(bool enableOOK);
+    //int16_t setOOK(bool enableOOK);
```

Comment:
Can you explain why this?

Reply:
The setOOK method was mixing some code related to RF69 with other code from CC1101. I just commented the method as a quick fix to be able to compile.

Now I updated the pull request and this method is available and working.